src/plugins/jingle/chat-header-notification.js

```diff
@@ -0,0 +1,45 @@
+import { CustomElement } from 'shared/components/element.js';
+import { _converse, api, converse } from "@converse/headless/core";
+import tpl_header_button from "./templates/header-button.js";
+import { JINGLE_CALL_STATUS } from "./constants.js";
+
+import './styles/jingle.scss';
+const { Strophe, $msg } = converse.env;
+
+export default class CallNotification extends CustomElement {
+    
+    static get properties() {
+        return {
+            'jid': { type: String },
+        }
+    }
+
+    initialize() {
+        this.model = _converse.chatboxes.get(this.jid);
+        this.listenTo(this.model, 'change:jingle_status', () => this.requestUpdate());
+    }
+    
+    render() {
+        return tpl_header_button(this);
+    }
+
+    endCall() {
+        const jingle_status = this.model.get('jingle_status');
+        if ( jingle_status === JINGLE_CALL_STATUS.OUTGOING_PENDING || jingle_status === JINGLE_CALL_STATUS.ACTIVE) {
```

Comment:
The stanza with a `retract` element is only sent when you want to cancel an outgoing pending call, not an active call.

To stop an active call, a different stanza is sent.
https://xmpp.org/extensions/xep-0353.html#finish

Reply:
Thanks for pointing it out, corrected it